### PR TITLE
add baked clay 

### PR DESCRIPTION
**Pull request by berengma**
check if baked clay mod is present and add all baked_clay nodes to default chisel program

### CODE REVIEW

File: chisel.lua [hunk 1]
```diff
@@ -47,6 +47,31 @@ local default_material = {
...
+      {"violet", "Violet"},
+      {"brown", "Brown"},
+      {"pink", "Pink"},
+      {"dark_grey", "Dark Grey"},
+      {"dark_green", "Dark Green"},
+   }
+
+   for _, clay in pairs(clay) do
+      table.insert(default_material,{"bakedclay:"..clay[1] , "baked_clay_" .. clay[1], clay[2] .. " Baked Clay"})
+   end
```
Comment (SmallJoker) [inline chisel.lua]: A majority of the lines are indented using tabs. Please use tabs for your changes to not make it an even larger mess.

File: chisel.lua [hunk 2]
```diff
@@ -157,8 +182,13 @@ local function chiselcut(pos,user,node)
 	     
 	     if chisel.materials[i][1] == chisel.active[name] then	     
 	      if node.name == chisel.materials[i][3] and chisel.materials[i][4] == chisel.materials[chisel.program[name]][4] then
-		
-		minetest.set_node(pos, {name=chisel.materials[i][1]..":"..chisel.materials[i][2].."_"..chisel.materials[i][4], param2=minetest.dir_to_facedir(user:get_look_dir())})
+		local stack = ItemStack(chisel.materials[i][1]..":"..chisel.materials[i][2].."_"..chisel.materials[i][4])
+		if stack:is_known() then
+		   minetest.set_node(pos, {name=chisel.materials[i][1]..":"..chisel.materials[i][2].."_"..chisel.materials[i][4], param2=minetest.dir_to_facedir(user:get_look_dir())})
```
Comment (SmallJoker) [inline chisel.lua]:
```suggestion
		minetest.set_node(pos, {
			name = chisel.materials[i][1] .. ":" .. chisel.materials[i][2] .. "_" .. chisel.materials[i][4],
			param2 = minetest.dir_to_facedir(user:get_look_dir())
		})
```

File: chisel.lua (hunk 1 above)
Comment (berengma) [reply chisel.lua]: per default "kate" editor uses spaces as identation. Do not ask me why. I had to search in the prefs to find that. I hope I will find some time this weekend

**Review by berengma — COMMENTED** [review]
all spaces replaced with tabs now.

splitting that minetest.set_node .... in multiple lines, what should that be good for ?
I find it more readable and understandable as it is, especially if you use big screens like me :)

**Comment by SmallJoker** [comment]
@berengma The suggested change is primary for easier reading, so that different table values are on a separate line. In Minetest it's clear what code style to use (80 character limit), but for Minetest-mods that depends on each mod. If you insist on keeping the current notation, please at least fix the indention of these few lines.
PS: Not everybody has 16:9 or even wider aspect ratios.

**Comment by berengma** [comment]
@SmallJoker indention is already fixed with latest commit

**Pull request merged**